docs/modules/user-guide/partials/proc_refering-to-a-parent-devfile-in-a-devfile.adoc

```diff
@@ -46,7 +46,7 @@ parent:
 +
 .Parent identified by a Kubernetes resource
 ====
-Using a Kubernetes resource name, namespace and selector if it has been deployed on a Kubernete cluster.
+Using a Kubernetes resource name and namespace if it has been deployed on a Kubernete cluster.
```

Comment:
For `Example 2. Parent referred by uri`

remove the id from the snippet. There can be only one of id/uri/kubernetes at any time

Reply:
I would also remove the version numbers from the `id` field since devfile v2 registry doesnt support this atm. What do you think @elsony?

Reply:
+1